hack/prom-rule-ci/verify-rules.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -xe
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2020 Red Hat, Inc.
+#
+
+readonly PROM_IMAGE="docker.io/prom/prometheus:v2.15.2"
+
+function main() {
+    local prom_spec_dumper="${1:?}"
+    local target_file
+
+    target_file="$(mktemp --tmpdir -u tmp.prom_rules.XXXXX)"
```

Comment:
I guess we would also have to clean them up?

Reply:
Done